src/lib.rs

```diff
@@ -156,7 +159,7 @@ fn on_remove_unfocus(mut world: DeferredWorld, context: HookContext) {
 }
 
 /// Sent when a text input submits its text
-#[derive(Event)]
+#[derive(Message)]
 pub struct TextSubmitEvent {
```

Comment:
Open question: what do we do with all `FooEvent`s that are now `Message`s? I didn't rename them without your opinion first, but Cart says we should avoid suffixes entirely, so IMO this should be `Message<TextSubmit>` in the present tense, just like `Event<Press>`

Reply:
Yep removing the suffix seems fine.

Reply:
I'll add a deprecated alias for the old one then to ease migration :)